app/views/courses/salary.html.erb

```diff
@@ -0,0 +1,68 @@
+<%= content_for :page_title, "#{course.name_and_code} - Course length and salary" %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(provider_course_path(course.provider.provider_code, course.course_code)) %>
+<% end %>
+
+<h1 class="govuk-heading-xl">
+  <span class="govuk-caption-xl"><%= course.name_and_code %></span>
+  Course length and salary
+</h1>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for course, url: publish_provider_course_path(course.provider.provider_code, course.course_code), method: :post do |f| %>
+      <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <h2 class="govuk-fieldset__heading">Course length</h2>
+          </legend>
+            <div class="govuk-radios" data-module="radios">
```

Comment:
Should this be a partial that's shared with fees page?

Reply:
Good shout - Done.